### PR TITLE
Update free-courses-hi.md

### PR DESCRIPTION
**Pull request by wildlucifer**
Added a playlist of R programming.

## What does this PR do?
A youtube playlist for R programming in Hindi.

### Description
Complete tutorial to learn  R programming.
### Why is this valuable (or not)?
It is complete resource for learning R programming from beginner to advanced.
### How do we know it's really free?
It is a youtube channel.
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by LuigiImVector** [comment]
resolve linter problems

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -376,7 +376,7 @@
 
 
 ### R
-
+* [Complete R Programming Tutorial for Beginners to Pro [HINDI]](https://www.youtube.com/playlist?list=PLjVLYmrlmjGdmPrz0Lx7smkd0qIKHInOF) - WsCube Tech
```
Comment (bugbunnyy) [inline courses/free-courses-hi.md:379]:
```suggestion

* [Complete R Programming Tutorial for Beginners to Pro [HINDI]](https://www.youtube.com/playlist?list=PLjVLYmrlmjGdmPrz0Lx7smkd0qIKHInOF) - WsCube Tech
```

**Review by bugbunnyy — CHANGES_REQUESTED** [review]

**Comment by LuigiImVector** [comment]
solve conflicts

**Comment by shrn01** [comment]
Closing the PR as there are no changes

**Pull request closed without merging**